robolectric/src/test/java/org/robolectric/internal/bytecode/AndroidTranslatorClassInstrumentedTest.java

```diff
@@ -94,6 +97,7 @@ public int getColor() {
   }
 
   @SuppressWarnings({"UnusedDeclaration"})
+  @Instrument
```

Comment:
I think there were just two instances of where additional classes needed to be instrumented, which is here for tests.

There were two cases (TestRunnerSequenceTest+CustomRobolectricTestRunnerTest where there were classes that needed to be explicitly excluded from being loaded by the InstrumentingClassloader the CustomRobolectricTestRunnerTest and TestRunnerSequenceTest.StateHolder, although I'm not really clear why they specifically would be (I was just refactoring what was already there)...but I'd like to understand that more.

There were three others in InstrumentingClassLoaderTest which were just testing different configurations, again, I think these should be different configurations of the same type, rather than subclasses.